selftests/unit/test_archive.py

```diff
@@ -5,6 +5,8 @@
 import sys
 import random
 
+from six.moves import xrange
```

Comment:
Well, here you're keeping the py2 syntax, while in previous commit (izip/zip) you changed the syntax to py3. I'd like to see a standard approach on that. Ok, zip is not present in py2 and xrange is not present in py3. Still, for consistency, I think we should move all the syntax to be py3 native and py2 compatible.
In that case, for this change, we should use range and have a `range = xrange` for py2. How does that sound?

Reply:
OK, fair enough.  This means `from six.moves import xrange as range`, right?

Reply:
yep